bindings/experimental/distrdf/python/DistRDF/Backends/Spark/Backend.py

```diff
@@ -161,3 +168,40 @@ def make_dataframe(self, *args, **kwargs):
         npartitions = kwargs.pop("npartitions", self.optimize_npartitions())
         headnode = HeadNode.get_headnode(npartitions, *args)
         return DataFrame.RDataFrame(headnode, self)
+
+    @staticmethod
+    def RunGraphs(proxies, numthreads=4):
```

Comment:
_if_ we stick with having a default `numthreads`, I think it should only be specified in the main `RunGraphs` definition in `DistRDF/__init__.py`, and the implementations in the different backends should require the parameter (so we have only one place where the magic number is specified).

also should `numthreads` be called `concurrentruns` also here for consistency?

Reply:
I was unsure about naming concurrentruns everywhere because with Spark we spawn multiple threads, with other backends might have some other mechanism. But since the common logic will always be submitting concurrent jobs/runs, I think it is fine